04.android/06.solving-problems/05.har/docs.en.md

```diff
@@ -7,43 +7,32 @@ visible: true
 ---
 
 * [What are HAR files?](#whatare)
-
 * [How to get HAR files?](#howtoget)
-
 * [How to analyze HAR files?](#howtoanalyze)
 
 <a id="whatare"></a>
 ## What are HAR files?
-
 The HAR (HTTP ARchive) format is a JSON-formatted archive file for logging of a web browser's interaction with a site. The specifications of HAR format defines an archival data for HTTP transactions that used by a browser to export detailed information about web pages it loads. More detailed description of HAR file format and its specification you can find at [softwareishard.com](www.softwareishard.com/blog/har-12-spec) website.
 <a id="howtoget"></a>
 ## How to get HAR files?
-
-Sometimes we need to analyze it to block specific ads that are difficult to reproduce for some reason.
-
+Sometimes we need to analyze it to block specific ads that are difficult to reproduce for some reason. To get HAR files follow these steps:
 1. Open AdGuard and go to "Settings".
-
 2. Choose "Advanced" in the menu.
-
 3. Choose "Low Level Settings"
-
 4. Activate "pref.har.capture" (you will have to restart the protection).
-
 5. Now reproduce the problem - open the app, perform the necessary actions to make the ad appear.
-
 6. Now turn "pref.har.capture" off again.
-
 7. Go to app cache folder and find the .har files. They are usually located in
-
-  **/storage/emulated/Android/data/com.adguard.android/cache/har**
-
+```**/storage/emulated/Android/data/com.adguard.android/cache/har**```
```

Comment:
Ask @Alex-302 what's preferred way of exporting a HAR file.

Reply:
He said it doesn't matter, whichever is more convenient for users. Just exporting the logs is fine